assets/scss/base/_icons.scss

```diff
@@ -114,6 +114,29 @@ Styleguide Icons.Exclamation
 }
 
 /*
+Tick
+
+The tick icon is used to highlight actioned content to the user.
+
+Markup:
+<i class="tick"></i>
```

Comment:
I think they need to have the same information conveyed - which they would miss if the element was hidden from them. For example if this was used to show items in a list were completed.
I guess it depends on if we want this to be a self-contained item with meaning, or if it's assumed it will be used wisely in conjunction with hidden text and this is purely for visual treatment.